st2common/st2common/config.py

```diff
@@ -139,6 +139,17 @@ def register_opts(ignore_errors=False):
     ]
     do_register_opts(api_opts, 'api', ignore_errors)
 
+    # Key Value store options
+    keyvalue_opts = [
+        cfg.BoolOpt('enable_encryption', default=True,
+                    help='Allow encryption of values in key value stored qualified as "secret".'),
+        cfg.StrOpt('encryption_key_path', default='conf/st2_kvstore_demo.crypto.key.json',
```

Comment:
Default path should probably point to some safer location and we should override this setting in the tests.